Eventing Docs updates for 6.6 (#1474)
couchbase/docs-server: modules/eventing/pages/eventing-example-data-enrichment.adoc

  ** For the *Settings* option, use the default values.
 ** For the *Bindings* option, add two bindings.

>  *** For the first binding, select "bucket alias", specify *src* as the "alias name" of the bucket, and select *source* as the associated bucket, and select "read only".

Reviewer: ```suggestion
 *** For the first binding, select "bucket alias", specify *src* as the "alias name" of the bucket, select *source* as the associated bucket, and select "read only".
```

>  *** For the second binding, select the "bucket alias", specify *tgt* as the "alias name" of the bucket, and select *target* as the associated bucket, and select "read and write".

Reviewer: ```suggestion
 *** For the second binding, select "bucket alias", specify *tgt* as the "alias name" of the bucket, select *target* as the associated bucket, and select "read and write".
```

Consistency with previous step

>  ** After configuring your settings your screen should look like:

Reviewer: ```suggestion
 ** After configuring your settings the *ADD FUNCTION* dialog should look like this:
```
